README.md

```diff
@@ -77,53 +96,85 @@ perl table_annovar.pl input/test_VEP.vcf hg38 --buildver hg38 --out test_VEP --r
 python InterVar.py -b hg38 -i input/test_VEP.vcf --input_type=VCF -o test_VEP
 ```
 4. Run [AutoPVS1 v2.0](https://github.com/JiguangPeng/autopvs1/releases/tag/v2.0).
```

Comment:
Can one of you @naqvia or @rjcorb create a new issue that this will need to be changed since AutoPVS1 is not working out of the box. We might either have to supply a copy here or point to our copy in the KF repo.